### DISCUSSION TITLE
milvus异常【SOS】  fail to search on all shard leaders

### DISCUSSION THREAD

**Discussion by RongHouchao** [milvus-io/milvus#23203] [Q&A and General discussion]
MilvusOperateException(status=500, msg=fail to search on all shard leaders, err=All attempts results:
attempt #1:fail to get shard leaders from QueryCoord: no replica available
attempt #2:fail to get shard leaders from QueryCoord: no replica available
attempt #3:fail to get shard leaders from QueryCoord: no replica available
attempt #4:fail to get shard leaders from QueryCoord: no replica available
attempt #5:fail to get shard leaders from QueryCoord: no replica available
attempt #6:fail to get shard leaders from QueryCoord: no replica available
attempt #7:fail to get shard leaders from QueryCoord: no replica available
attempt #8:context deadline exceeded

**Comment by yhmo** [comment]
Similar issues:
https://github.com/milvus-io/milvus/issues/18205
https://github.com/milvus-io/milvus/issues/22383

Which version of the milvus?
Could you provide more details such as reproduce steps, log of querycoord/querynode, etc.
